scripts/dockerfiles/run.sh

```diff
@@ -78,9 +78,33 @@ service rsyslog start
 cron
 
 # Run custom post config role scripts
+# Run any custom scripts other than pre-start
 {{ range $script := .role.PostConfigScripts}}
+{{ if not (is_pre_start $script) }}
+    echo bash {{ if not (is_abs $script) }}/opt/hcf/startup/{{ end }}{{ $script }}
     bash {{ if not (is_abs $script) }}/opt/hcf/startup/{{ end }}{{ $script }}
 {{ end }}
+{{ end }}
+
+# Run all the scripts called pre-start, but ensure consul_agent/bin/pre-start is run before others.
+# None of the other pre-start scripts appear to have any dependencies on one another.
+# Use Perl to sort consul_agent/bin/pre-start before others.
+# (Perl's sort is stable, not that it matters for `find' output.)
+function sorted-pre-start-paths()
```

Comment:
Possibly. I gave myself two constraints: run `find' only once, and don't use a temporary file to store results.  I'll try something that doesn't introduce a dependency on Perl.